### DISCUSSION TITLE
How do I update the dependencies for `"_project_#@lhci#cli#lighthouse#intl-messageformat"`?

### DISCUSSION THREAD

**Discussion by hirehamir** [GoogleChrome/lighthouse-ci#943] [Q&A] [answered]
The motivation behind this is that I'd like to update `intl-messageformat-parser`.

It's not listed in [`package.json`](https://github.com/GoogleChrome/lighthouse-ci/blob/main/package.json), and `yarn` says

```
=> Found "intl-messageformat-parser@1.8.1"
info Reasons this module exists
   - "_project_#@lhci#cli#lighthouse#intl-messageformat" depends on it
   - Hoisted from "_project_#@lhci#cli#lighthouse#intl-messageformat#intl-messageformat-parser"
```

**Comment by adamraine** [comment, marked as answer]
It's a dependency of Lighthouse which is a dependency of `@lhci/cli`

**Reply by hirehamir** [reply]
Thanks for clarifying!

For future people reading this, it looks like `intl-messageformat-parser` comes from https://github.com/GoogleChrome/lighthouse, which [`packages/cli`](https://github.com/GoogleChrome/lighthouse-ci/tree/main/packages/cli) uses.

https://github.com/GoogleChrome/lighthouse/blob/268e3af73d81fb365ee2a46351591e7085bab1d7/package.json#L155
